images/clickhouse/Dockerfile

```diff
@@ -40,6 +40,7 @@ COPY staging/images/{{ instance_name }}/config/clickhouse-keyring.gpg /usr/share
 RUN cd ${CH_TMP_DIR} && \
     pip3 install -r requirements.txt && \
     mkdir -p /etc/apt/sources.list.d && \
+    apt-get update -qq && \
     echo "deb [signed-by=/usr/share/keyrings/clickhouse-keyring.gpg] https://packages.clickhouse.com/deb stable main" | tee /etc/apt/sources.list.d/clickhouse.list && \
     apt-get update -qq && \
```

Comment:
If I'm not mistaken the repository index was already regenerated. And this fix is not needed. By the way we have `apt-get update` below.

Reply:
yes, looks like.
Will close the pr.